plugins/auth-backend/src/lib/catalog/CatalogIdentityClient.ts

```diff
@@ -106,8 +103,9 @@ export class CatalogIdentityClient {
       'metadata.namespace': ref.namespace,
       'metadata.name': ref.name,
     }));
+    const { token } = await this.tokenManager.getToken();
     const entities = await this.catalogApi
-      .getEntities({ filter })
+      .getEntities({ filter }, { token })
```

Comment:
Note: a token wasn't being passed here, which may be a good thing considering it would've been hard to depend on the token behavior that we're breaking in this client. If an adopter has set up backend to backend authentication then I'd assume they aren't using this for identity resolution